Source/WebCore/css/parser/CSSParserToken.cpp

```diff
@@ -751,12 +752,17 @@ void CSSParserToken::serialize(StringBuilder& builder, const CSSParserToken* nex
     case BadUrlToken:
         builder.append("url(()"_s);
         break;
-    case WhitespaceToken: {
+    case NonNewlineWhitespaceToken: {
         auto count = mode == SerializationMode::CustomProperty ? m_whitespaceCount : 1;
         for (auto i = 0u; i < count; ++i)
             builder.append(' ');
         break;
     }
+    case NewlineToken: {
+        if (mode == SerializationMode::CustomProperty)
+            builder.append('\n');
```

Comment:
We need to append a space here if it's not a custom property. 
Otherwise we could fail to round-trip it seems :
2 identifiers like `foo\nbar` would serialize to a single identifier `foobar` which is wrong.

Reply:
Maybe we can add a test for this